src/pages/Crew/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import { useSwipeable } from "react-swipeable";
+import {
+  CarouselWrapper,
+  PaginationDot,
+  Inner,
+  PaginationDotsWrapper,
+} from "./Carousel.styles";
+
+interface CarouselProps {
+  children: React.ReactElement[];
+}
+
+export default function Carousel({ children }: CarouselProps) {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  const translateValue = activeIndex * 100;
```

Comment:
w przypadku występowania tzn 'magic numbers' warto dać w kodzie znać np. komentarzem skąd akurat '100'. Jeśli to np procenty, to wystarczy dodać w komentarzu '%' i będzie wiadomo :D